components/shared/filters/SidebarFilter.vue

```diff
@@ -4,6 +4,10 @@
       <EventTypeFilter v-if="isCollectionActivityTab" expanded fluid-padding />
       <StatusFilter v-else expanded fluid-padding />
       <PriceFilter v-if="!isCollectionActivityTab" fluid-padding />
+      <PopularCollections
+        v-if="!isCollectionActivityTab && !isCollectionDetail"
```

Comment:
maybe it will be clearer to choose where is displayed
rather then choosing where it isn't